itou/www/dashboard/forms.py

```diff
@@ -21,6 +21,7 @@ def __init__(self, *args, **kwargs):
         editor = kwargs.pop("editor")
         super().__init__(*args, **kwargs)
 
+        # Only precribers and employers can edit the job seeker's email here under certain conditions
         if not self.instance.is_job_seeker or not editor.can_edit_email(self.instance):
```

Comment:
Tu es sûr de cette condition ? Vu la description j’ai l’impression que
 - si on est `job_seeker`, on ne modifie jamais l’email
 - si on est pas `job_seeker`, il faut tester `editor.can_edit_email`
si c’est bien ça, j’ai l’impression qu’il faudrait plutôt:
```suggestion
        if self.instance.is_job_seeker or not editor.can_edit_email(self.instance):
```

Reply:
si c’est bien ça il y a peut-être un test à ajouter ?

Reply:
ok, je vois, j’avais pas en tête la condition sur les prescripteurs et employeurs. On peut peut-être ajouter un commentaire à ce sujet pour clarifier ?
```
# Noboby can edit its own email.
# Only prescribers and employers can edit the job seeker's email here under certain conditions
```

Reply:
En effet, ça ne mange pas de pain !